GAE/src/locale/ui-strings.properties

```diff
@@ -333,7 +333,7 @@ _number = Number
 _number_of_question_groups = Number of question groups
 _number_of_questions = Number of questions
 _number_of_users = Number of users
-_ok = OK
+_ok = Delete
```

Comment:
Rather than changing the text here, (this placeholder may be used by other elements in the dashboard) use a `_delete` placeholder in the dialog control button

Reply:
the challenge is that dialog control button is set in one file, application.handlebars from what i have seen so far, in order to change the text all i had to do was change the placeholder there, say from _ok to _delete, but that still changes it to delete everywhere.

Reply:
> but that still changes it to delete everywhere.

yes. and thats ok.  All I meant was that if we edit the _ok placeholder, it may be used in other places where its inappropriate to have the text "Delete".  So rather use the _delete placeholder.

Reply:
i've replaced _ok with _delete, changed _ok back to OK, but it still doesn't deal with the issue since its one html file for the dialogues